TeachingPlatform/users/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+
+from TeachingPlatform.users.views import HomeView, TeachersView, AddTeachersView, StudentsView, DeleteStudentView, DeleteTeacherView, \
+    EditTeacherView
+
+urlpatterns = [
+    path("", HomeView.as_view(), name='home_temp'),
+    path("teachers/", TeachersView.as_view(), name='teachers_list'),
+    path("add_teachers/", AddTeachersView.as_view(), name='teachers_add'),
+    path("students/", StudentsView.as_view(), name='students_list'),
+    path("students_del/<int:student_id>", DeleteStudentView.as_view(), name='students_del'),
+    path("teachers_del/<int:teacher_id>", DeleteTeacherView.as_view(), name='teachers_del'),
```

Comment:
All variables and view names should not contain abbreviations, to make them easier to read and understand. In this case, the name of the view should be "teacher_delete". The same applies to all other shortened names, like "students_del" :smile: